src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -278,6 +289,23 @@ internal void CreateExitModal(ExitGuide exitGuide)
             exitGuideWindow.IsOpen = true;
         }
 
+        /// <summary>
+        /// Shows/Hides the Popup based in if the DynamoView is Active or not
+        /// </summary>
+        /// <param name="isActive"></param>
+        internal void ManagePopupActivation(bool isActive)
+        {
+            switch(currentGuideState)
```

Comment:
Can we resue the events somehow, for example track the last triggered events, etc.

Reply:
@RobertGlobant20 now I'm using the events from the class GuideFlowEvents, are you referring to those events?
https://github.com/DynamoDS/Dynamo/pull/13002/commits/d382ee7fbcf6a3ccf04ffe5477e07e8d13d73fd2